src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDeposit.scala

```diff
@@ -239,8 +243,21 @@ object AddDatasetMetadataToDeposit {
     createSchemedMetadata(dataset, composedSubjectFields, "DC_SUBJECT", "DC_SUBJECT_SCHEME")
   }
 
+  /*
+    q   l   t     valid
+    1   1   1     0
+    1   1   0     1
+    1   0   1     1
+    1   0   0     0
+    0   1   1     0
+    0   1   0     1
+    0   0   1     1
+    0   0   0     1
+
+    conclusie: mag niet tegelijk DCX_RELATION_LINK en DCX_RELATION_TITLE hebben
```

Comment:
After some thought, I brought back the truth table. Seems to be a good addition as documentation.